lib/src/main/scala/spinal/lib/bus/amba4/axi/Axi4Channel.scala

```diff
@@ -533,8 +544,11 @@ case class FormalAxi4Record(val config: Axi4Config, maxStrbs: Int) extends Bundl
   def checkLen(): Bool = {
```

Comment:
Alternative syntax to avoid to have to manualy name signal is : 

```scala
def checkLen() = new Composite(this, "checkLen"){ ... ; val value = xxx}.value
```

Reply:
> Alternative syntax to avoid to have to manualy name signal is :
> 
> ```scala
> def checkLen() = new Composite(this, "checkLen"){ ... ; val value = xxx}.value
> ```

Good.